README.md

```diff
@@ -146,3 +146,49 @@ let response = await api.updateBug(123456, {
 ```
 
 Returned value is same as described in Bugzilla [docs](https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#update-bug).
+
+## Retrieving attachments by ID
+
+```javascript
+// .getAttachment() accepts one parameter, ID of attachment, as number
+let attachment = await api.getAttachment(123456);
+```
+
+Return value is Attachment object.
+
+## Retrieving all attachments of bug
+
+```javascript
+// .getBugsAttachments() accepts one parameter, ID of bug, as number
+let attachments = await api.getBugsAttachments(123456);
+```
+
+Return value is array of Attachment objects.
+
+## Creating attachments
+
+```javascript
+let attachment = await api.createAttachment(123456, {
+  ids: [123456, 123457];
+  data: "base64-content";
+  file_name: "Attachment name";
+  summary: "Attachment summary";
+  content_type: "text/plain";
+  is_private?: false;
+});
+```
+
+Returned value is an array of IDs of the newly-created attachments.
+
+## Updating attachments
+
+Example of changing content type of attachment #123456:
```

Comment:
Is it possible to change anything other than content type with this API?

Reply:
In theory it should be possible to change more than only content type - [docs](https://bugzilla.readthedocs.io/en/latest/api/core/v1/attachment.html#update-attachment).